0003-messaging-ipc/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: Messaging IPC
+shortname: 3/MIPC
+status: raw
+contributors: Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
```

Comment:
I'm not sure, can you please help me decide? After an RFC is drafted by a dev (contributor/editor?), it is reviewed by other devs (contributors/editors?), and then techwriter makes a final review (editor?).
In my eyes, the first dev should be author, other devs -- contributors, and the techwriter should be editor. But I'm not sure which process we follow here.